drivers/hmis/lib/form_data/tarrant/fragments/patches/sexual_orientation.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "link_id": "demographics",
+    "append_items": [
+      {
+        "type": "CHOICE",
+        "link_id": "sexual_orientation",
+        "text": "Sexual Orientation",
+        "mapping": {
+          "custom_field_key": "sexual_orientation"
+        },
+        "component": "DROPDOWN",
+        "pick_list_options": [
+          {
+            "code": "Straight"
```

Comment:
Note not to forget to update this dropdown with the correct values

Reply:
```
Heterosexual
Gay
Lesbian
Questioning/unsure
Bi-sexual
Pan-sexual
Client Doesn't Know
Client refused
Data not collected
```
these will match the migrated data exactly so make sure you use the same capitalization and characters. thanks!

Reply:
thank you!